modules/pipeline-manifest/Makefile

```diff
@@ -181,6 +185,32 @@ pipeline-manifest/_promote: %_promote: %_clone
 	@cd $(PIPELINE_MANIFEST_DIR); $(GIT) commit -am "Promote from $(PROMOTE_FROM) to $(PROMOTE_TO)" --quiet
 	@cd $(PIPELINE_MANIFEST_DIR); $(GIT) push
 	
+.PHONY: pipeline-manifest/_osci_to_quay
+# Copy image from OpenShift CI registry to quay.io
+pipeline-manifest/_osci_to_quay:
+	$(call assert-set,DOCKER_USER)
+	$(call assert-set,DOCKER_PASS)
+	@echo Copy OpenShift CI image to Quay
+	@echo .. Getting image remote for component $(PIPELINE_MANIFEST_COMPONENT)
+	@$(JQ) -r '$(PM_IMAGE_REMOTE_QUERY)' $(PIPELINE_MANIFEST_DIR)/$(PIPELINE_MANIFEST_FILE_NAME).json > IMAGE_REMOTE
+	@$(JQ) -r '$(PM_GIT_SHA_QUERY)' $(PIPELINE_MANIFEST_DIR)/$(PIPELINE_MANIFEST_FILE_NAME).json > GIT_SHA
+	@echo Image remote is ; cat IMAGE_REMOTE
+	@if grep '^registry\.svc\.ci\.openshift\.org/open-cluster-management$$' IMAGE_REMOTE ; \
+	then \
+		echo .... Copying from OpenShift CI to Quay ; \
+		OSCI_REMOTE="`cat IMAGE_REMOTE`/$(PIPELINE_MANIFEST_RELEASE_VERSION):$(PIPELINE_MANIFEST_COMPONENT)" ; \
+		QUAY_REMOTE="$(PIPELINE_MANIFEST_REMOTE_REPO)/$(PIPELINE_MANIFEST_COMPONENT):`cat GIT_SHA`" ; \
```

Comment:
We generally tag the image in quay with the release in front of it as well - probably not critical, and you'd need to find a way to sniff out the Z-release version number.